missioncontrol/home/models.py

```diff
@@ -350,11 +350,10 @@ def clean(self):
         not_stale = (Q(scheduled_on_sat=True) |
                      Q(scheduled_on_gs=True) |
                      Q(is_desired=True))
-        same_sat = Q(satellite=self.satellite)
         same_gs = Q(groundstation=self.groundstation)
         different_id = ~Q(uuid=self.uuid)
         overlapping_pass_qs = Pass.objects.filter(
-            same_sat & same_gs & different_id & not_stale &
+            same_gs & different_id & not_stale &
```

Comment:
this should be `same_sat` *or* `same_gs`

Reply:
also an aside, but we can make this a whole lot more performance now that we are on postgres.

Reply:
> this should be `same_sat` _or_ `same_gs`

Good point, fixed up :)


> also an aside, but we can make this a whole lot more performance now that we are on postgres.

I'm not clear how?

Reply:
https://docs.djangoproject.com/en/2.1/ref/contrib/postgres/fields/#datetimerangefield
(no need to over-optimize now..)